playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -71,8 +71,9 @@
 
 - name: Copy site to Shipyard
   become: yes
-  delegate_to: '{{ inventory_hostname }}'
-  synchronize:
+  copy:
+    remote_src: yes
+    mode: "preserve"
```

Comment:
[2019-07-29T09:45:33.816Z]   msg: 'Remote copy does not support recursive copy of directory: /opt/airship/treasuremap/peggles'

Reply:
maybe due to  `remote_src supports recursive copying as of version 2.8.` didnt we had 2.8 already in the CI?

Reply:
no we dont, 2.7.8 still on CI:

```
ccp-ci-launcher:~ # ansible --version
ansible 2.7.8
```